src/Nancy/Json/Json.cs

```diff
@@ -118,10 +118,11 @@ public static bool IsJsonContentType(string contentType)
             var contentMimeType = contentType.Split(';')[0];
 
             return contentMimeType.Equals("application/json", StringComparison.OrdinalIgnoreCase) ||
-                   contentMimeType.StartsWith("application/json-", StringComparison.OrdinalIgnoreCase) ||
-                   contentMimeType.Equals("text/json", StringComparison.OrdinalIgnoreCase) ||
-                  (contentMimeType.StartsWith("application/vnd", StringComparison.OrdinalIgnoreCase) &&
-                   contentMimeType.EndsWith("+json", StringComparison.OrdinalIgnoreCase));
+            contentMimeType.StartsWith("application/json-", StringComparison.OrdinalIgnoreCase) ||
+            contentMimeType.Equals("text/json", StringComparison.OrdinalIgnoreCase) ||
+            (contentMimeType.StartsWith("application/vnd", StringComparison.OrdinalIgnoreCase) &&
+            contentMimeType.EndsWith("+json", StringComparison.OrdinalIgnoreCase)) ||
+            contentMimeType.Equals("application/merge-patch+json");

```

Comment:
so remove

```
(contentMimeType.StartsWith("application/vnd", StringComparison.OrdinalIgnoreCase) &&
contentMimeType.EndsWith("+json", StringComparison.OrdinalIgnoreCase)) ||
contentMimeType.Equals("application/merge-patch+json")
```

and just make it `contentMimeType.EndsWith("+json", StringComparison.OrdinalIgnoreCase)`